fetch.bs

```diff
@@ -4283,32 +4307,50 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
 
       <!-- cache hit -->
       <ol>
-       <li><p>If <var>storedResponse</var> requires validation (i.e., it is not fresh), then set the
-       <var>revalidatingFlag</var>.
-
-       <li>
-        <p>If the <var>revalidatingFlag</var> is set and <var>httpRequest</var>'s
-        <a for=request>cache mode</a> is neither "<code>force-cache</code>" nor
-        "<code>only-if-cached</code>", then:
+       <li><p>If <a for=request>cache mode</a> is "<code>default</code>" and
```

Comment:
The indenting is wrong here. If an `<li>` has multiple nested elements you need to put the `<p>` on a newline (and indent with an extra space).

Reply:
done